src/prefect/futures.py

```diff
@@ -297,6 +298,41 @@ def result(
             ) from exc
 
 
+DoneAndNotDoneFutures = collections.namedtuple("DoneAndNotDoneFutures", "done not_done")
+
+
+def wait(pfs: List[PrefectFuture], timeout=None) -> DoneAndNotDoneFutures:
```

Comment:
nit: I was confused about the purpose of `pfs` on first read.
```suggestion
def wait(futures: List[PrefectFuture], timeout=None) -> DoneAndNotDoneFutures:
```